TSC-Charter.md

```diff
@@ -56,14 +56,23 @@ one-fourth of the TSC membership shares an employer, then the situation
 must be immediately remedied by the resignation or removal of one or more
 TSC members affiliated with the over-represented employer(s).
 
-The TSC members shall consist of Maintainers from Core Projects as
-defined in the project lifecycle document and Section 7.
+The TSC membership shall include (but is not limited to) Maintainers from Core
+Projects as defined in the project lifecycle document and Section 7.
 
 The TSC shall meet regularly using tools that enable participation by the
 community (e.g. weekly on a Google Hangout On Air, or through any other
 appropriate means selected by the TSC). The meeting shall be directed by
-the TSC Chairperson. Minutes or an appropriate recording shall be taken
-and made available to the community through accessible public postings.
+the TSC Chairperson. Responsibility for directing individual meetings may be
+delegated by the TSC Chairperson to any other TSC member. Minutes or an
+appropriate recording shall be taken and made available to the community
+through accessible public postings.
+
+In the case where an individual TSC member does not attend 60% of the regularly
+scheduled meetings held within a six month period, the member shall be asked to
+consider voluntarily resigning their membership. Should a member fail to attend
+100% of the meetings within a six month period, the member shall be
+automatically removed from the TSC. In such cases, the member may ask to
+continue as an invited observer to TSC meetings.
```

Comment:
If we do this I think it technically needs to have a clause that it cannot drop below 3 people.

Also, I feel 6 months could be a bit short? Unsure.